.github/workflows/linux-eic-shell.yml

```diff
@@ -223,20 +216,23 @@ jobs:
         network_types: "none"
         setup: install/setup.sh
         run: |
-          geoConverter -compact2gdml -input ${DETECTOR_PATH}/${DETECTOR}_${{ matrix.detector_config }}.xml -output ${{ matrix.detector_config }}.gdml
+          IFS=, read -a configs <<< "${{ needs.list-detector-configs.outputs.configs_csv }}"
+          for config in ${configs[@]} ; do
+            echo "::group::${config}" ;
+            geoConverter -compact2gdml -input ${DETECTOR_PATH}/${config}.xml -output ${config}.gdml ;
+            echo "::endgroup::" ;
+          done
     - uses: actions/upload-artifact@v3
       with:
         name: gdml
-        path: ${{ matrix.detector_config }}.gdml
+        path: ${{ github.workspace }}/*.gdml
```

Comment:
Why not just?
```suggestion
        path: *.gdml
```

Reply:
Not accepted in https://github.com/eic/epic/actions/runs/5494997781 with annotation in https://github.com/eic/epic/actions/runs/5494997781/workflow

Reply:
The problem isn't that it doesn't parse the `*` without quotes; it does when there is an explicit path prefix. Maybe with `./*.gdml`, but is that so much better?

Reply:
It's better in the sense that it doesn't refer to `github.workspace` consistently.

Reply:
```suggestion
        path: "*.gdml"
```